virt/virt-2-4-release-notes.adoc

```diff
@@ -28,9 +28,11 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 === Networking
 //Placeholder for new content.
 
-#CNV-5286 - SR-IOV Networking to provide a basic fast datapath option
+//CNV-5286 - SR-IOV Networking (no xref yet, PR not merged)
+* {VirtProductName} is now integrated with the {product-title} Single Root I/O Virtualization (SR-IOV) Operator. You can now attach virtual machines to SR-IOV networks in your cluster. 
 
-#CNV-5285 - MAC pool support was reintroduced to OpenShift virtualization. It is disabled by default and can be enabled per namespace.
+//CNV-5285 - MAC pool support 
+* xref:../virt/virtual_machines/vm_networking/virt-using-mac-address-pool-for-vms.adoc#virt-using-mac-address-pool-for-vms[MAC address pool] support has been reintroduced to {VirtProductName}. It is disabled by default in the cluster and can be enabled per namespace.
```

Comment:
This makes me wonder why it was removed and brought back. IDK this history on this one, but maybe s/has been reintroduced to/is supported in

Reply:
Yeah, there's a bit of history there but you're right, it's probably not too interesting. 
Updated as suggested.